nemo/core/connectors/save_restore_connector.py

```diff
@@ -133,7 +133,7 @@ def load_config_and_state_dict(
                 # Change current working directory to
                 os.chdir(tmpdir)
                 if override_config_path is None:
-                    config_yaml = os.path.join(tmpdir, self.model_config_yaml)
+                    config_yaml = self.model_config_yaml
```

Comment:
Why are we changing this?

Reply:
@lhb8125 might be able to answer this

Reply:
Since the working dir has changed to `tmpdir `by `os.chdir(tmpdir)`, we don't need to join the `tmpdir `with `model_config_yaml`.
And if a relative dir path of nemo checkpoint is specified in the config, the `config_yaml `will be `tmpdir/model_config_yaml`. However, the path is wrong since we already enter the `tmpdir`.